eos-update-server/eos-update-server.c

```diff
@@ -492,25 +516,48 @@ main (int argc, char **argv)
       return EXIT_DISABLED;
     }
 
-  repo = eos_updater_local_repo ();
-  server = eos_updater_repo_server_new (repo,
-                                        options.served_remote,
-                                        NULL,
-                                        &error);
-  if (server == NULL)
+  /* Set up the server and repositories. */
+  soup_server = soup_server_new (NULL, NULL);
```

Comment:
Well, reading the code for soup_server_new and g_object_new_valist makes me believe it should be fine, but I'm not going to get religious about it…